test/dommock/DOMMockBuilder.ts

```diff
@@ -736,6 +742,39 @@ export default class DOMMockBuilder {
       }
     };
 
+    const mockRTCRtpTransceiverWithCurrentDirection = class MockRTCRtpTransceiverWithCurrentDirection {
```

Comment:
I don't think you need to create the full clone of `MockRTCRtpTransceiver` but how about just setting it once?

```
if (mockBehavior.isUnifiedPlanSupported) {
  MockRTCRtpTransceiver.prototype.currentDirection = 'sendrecv';
}
```

Note that `RTCRtpTransceiver.prototype.hasOwnProperty('currentDirection');` will be `true` only when you explicitly set the prototype like the above.

Reply:
Good point, updated.